src/Akka.Persistence.Linq2Db.Benchmark.DockerTests/Docker/Linq2Db/DockerLinq2DbPostgreSQLJournalPerfSpec.cs

```diff
@@ -30,10 +30,12 @@ class = ""{typeof(Linq2DbWriteJournal).AssemblyQualifiedName}""
             connection-string = ""{connString}""
             provider-name = ""{ProviderName.PostgreSQL95}""
             use-clone-connection = true
-            tables.journal {{ 
-                auto-init = true
-                warn-on-auto-init-fail = false
-                table-name = ""testPerfTable"" 
+            auto-initialize = true
```

Comment:
This brings the setting back to where it usually is for all other Akka.Persistence implementations (in the plugin's `.journal` HOCON)